library/daloradius.conf.php.sample

```diff
@@ -68,6 +68,7 @@ $configValues['CONFIG_DB_TBL_DALONODE'] = 'node';
 $configValues['CONFIG_FILE_RADIUS_PROXY'] = '/etc/freeradius/proxy.conf';
 $configValues['CONFIG_PATH_RADIUS_DICT'] = '';
 $configValues['CONFIG_PATH_DALO_VARIABLE_DATA'] = '/var/www/daloradius/var';
+$configValues['CONFIG_MOUNTED_DIR'] = '';
```

Comment:
Can we update the code here so that the default is set to `/` like the original code?

I can see that you've added it here https://github.com/lirantal/daloradius/pull/223/files#diff-70470acfe6f3487fc2e87423d5a74ed532555180d2af4f0843c4ba9473f51916R314 with that `/` as a parameter to the function but it's a bit confusing. I assume you did that to preserve a default behavior. so perhaps we can have that default existing in the `get_hdd_freepsace()` function itself over here https://github.com/lirantal/daloradius/pull/223/files#diff-70470acfe6f3487fc2e87423d5a74ed532555180d2af4f0843c4ba9473f51916R157 ? This way, if `CONFIG_MOUNTED_DIR` config doesn't exist yet (which it won't in old codebases) then it can default to the other `/` directory.

Reply:
@crs26 ping, have you perhaps missed the comment :) ?

Reply:
@bruvv if they'd be slow to response and you'd want to make an update here (or in a new PR) I will be happy to merge it so we can get these fixes in for FR3.

Reply:
Wouldnt it be quicker just to merge it and then adjust line #71 yourself :P ?

Reply:
I can update this PR's code and merge it too, not an issue. What's the change that is missing and is everything else ok? :-)

Reply:
well if you can merge this into a development branch I can test it, as far as I can tell it looks good as is.

Reply:
@bruvv I'll merge to the main branch `master` - could you then run a quick check of the latest master branch to see that this didn't break anything?

Reply:
@bruvv that's all good and I wish you all well. Always prioritize yourself and family ❤️

Reply:
Sorry for a very late reply, I've been busy at my full time work. I'll push an update so it won't break the old codebases

Reply:
Hope everything's ok on your end, no rush. Would be great to update once you're ready :)